serving/ingress/pkg/reconciler/ingress/resources/route_test.go

```diff
@@ -543,3 +543,18 @@ func withHTTPSBackendService() ruleOption {
 		}}
 	}
 }
+
+func withArgoCDMetadata(ing *networkingv1alpha1.Ingress) {
+	annos := ing.GetAnnotations()
+	if annos == nil {
+		annos = map[string]string{}
+	}
+	labels := ing.GetLabels()
+	if labels == nil {
+		labels = map[string]string{}
+	}
+	annos[ArgoCDPrefix+"/key"] = "value"
```

Comment:
do the values here not matter? Or where is this coming from?

Reply:
Test only random values  to filter out.